modules/troubleshooting-common-problems-in-jupyter-for-administrators.adoc

```diff
@@ -18,17 +18,8 @@ If you have configured specialized user groups for {productname-short}, the user
 .Diagnosis
 Check whether the user is part of the default user group.
 
-. Find the names of groups allowed access to Jupyter.
-ifndef::upstream[]
---
-.. Log in to {openshift-platform} web console.
---
-endif::[]
-ifdef::upstream[]
-.. Log in to {productname-short} web console.
-endif::[]
-.. Click *User Management* -> *Groups*.
-.. Click the name of your user group, for example, `{oai-user-group}`.
+. In the {openshift-platform} web console, click *User Management* -> *Groups*.
+. Click the name of the default user group.
```

Comment:
What is meant by `default user group`? Do we refer to the user group used in the RHOAI product?

Reply:
The user groups are explained in this section here: https://access.redhat.com/documentation/en-us/red_hat_openshift_ai_self-managed/2.5/html/managing_users/adding-users_user-mgmt

Customers can configure their own user groups specific to RHOAI. For example, for a normal users (not admins), they can create a group called rhoai users, and add users to that group.

Reply:
Yeah, my point was more about whether it's clear for the reader what we mean by the term `default user group`. But looking also on lines above and it is probably okay from that context. Anyway, it's not a part of this issue.